src/main/java/no/unit/nva/amazon/s3/AbortMultipartUploadHandler.java

```diff
@@ -2,143 +2,103 @@
 
 
 import com.amazonaws.services.lambda.runtime.Context;
-import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.amazonaws.services.s3.AmazonS3;
 import com.amazonaws.services.s3.model.AbortMultipartUploadRequest;
 import com.amazonaws.services.s3.model.AmazonS3Exception;
-import com.google.gson.Gson;
-import com.google.gson.JsonSyntaxException;
-
-import java.util.Map;
-import java.util.Objects;
-import java.util.concurrent.ConcurrentHashMap;
-
-import static no.unit.nva.amazon.s3.Environment.ALLOWED_ORIGIN_KEY;
-import static no.unit.nva.amazon.s3.Environment.MISSING_ENV_TEXT;
-import static no.unit.nva.amazon.s3.Environment.S3_UPLOAD_BUCKET_KEY;
-import static no.unit.nva.amazon.s3.GatewayResponse.ACCESS_CONTROL_ALLOW_ORIGIN;
-import static no.unit.nva.amazon.s3.GatewayResponse.BODY_KEY;
-import static org.apache.http.HttpHeaders.CONTENT_TYPE;
-import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
-import static org.apache.http.HttpStatus.SC_INTERNAL_SERVER_ERROR;
-import static org.apache.http.HttpStatus.SC_NOT_FOUND;
+import no.unit.nva.amazon.s3.exception.InvalidInputException;
```

Comment:
I am not sure if s3 is a good package name. Better to have a name that shows what is the role of underlying classes (publication storage?) and not the technology that we are using. If it generic storage maybe we should name it something else.